core/api/jrpc/jrpc_server_impl.cpp

```diff
@@ -52,12 +53,13 @@ namespace kagome::api {
      * https://github.com/w3f/PSPs/blob/psp-rpc-api/psp-002.md#state_subscribestorage-pubsub
      */
     using Response = jsonrpc::Request;
-    using Value = jsonrpc::Value;
     using Fault = jsonrpc::Fault;
 
     JsonWriter writer;
     try {
-      Response response(std::move(method_name), from, Value(0));
+      // Prevent jsonrpc-lean from writing "id" field.
+      constexpr bool id = false;
```

Comment:
Hm, perhaps we can avoid adding a comment by renaming the constant something like this
```suggestion

      constexpr bool should_write_id = false;
```
?